yarn-project/aztec-rpc/package.json

```diff
@@ -33,10 +34,13 @@
   "dependencies": {
     "@aztec/acir-simulator": "workspace:^",
     "@aztec/circuits.js": "workspace:^",
+    "@aztec/ethereum": "workspace:^",
     "@aztec/foundation": "workspace:^",
     "@aztec/key-store": "workspace:^",
     "@aztec/noir-compiler": "workspace:^",
     "@aztec/types": "workspace:^",
+    "koa": "^2.14.2",
+    "koa-router": "^12.0.0",
```

Comment:
Not for this PR, but should we look into extracting the http services into a separate package? I was thinking we'll want to present the rpc-server (and maybe aztec-node as well?) as a library that can be embedded into something larger (eg into a wallet), and in that case the http api implementation is not needed.

I keep going back to the difference between application and module packages (see [here](https://github.com/AztecProtocol/aztec-packages/issues/204)), and I'm thinking that maybe rpc and node should be modules, and another package should wrap them into an app.

Or maybe I'm overcomplicating things. Let's roll with this for now and we can revisit this when we actually start composing the rpc server into larger apps.